etc/dev/jwt-provider.yaml

```diff
@@ -4,5 +4,4 @@ kid: 0
 keys:
   -
     id: 0
-    seed: glmfAFEfY2r3VzTocQ6e7Rip1Qac3uQA
-
+    seed: env://TRUSTY_JWT_SEED
```

Comment:
TRUSTY_JWT_SEED or $TRUSTY_JWT_SEED ?

Reply:
`env://` is internal implementation of config loader.
it could be `file://`